app/assets/javascripts/components/links/signal_icon.js.jsx.erb

```diff
@@ -0,0 +1,8 @@
+var SignalIcon = React.createClass({
+  render: function() {
+    var iconPath = this.props.path + '.png'
+    return (
+      <img src={'<%= image_url("icons/offer.png") %>'} className='panel-icon' />
```

Comment:
So weirdly, if you add .erb extension, 
1. it renders correctly
2. Errors out with javascript string concat (ie.  ` { '<%= image_url(\"' + iconPath + '\" %>'}   `